### PR TITLE
upgrade vulnerabilities angular 1.8.3

### PR DESCRIPTION
**Pull request by dskamiotis**
## What does this change?
Bumps AngularJS and related dependencies to 1.8.3.

As a result of amending <transclude> property in the element, this caused some styling of the buttons (particularly those with SVG's in them) to be misaligned) 
This required some further remedial work

In addition reducing noisy logging at he the start of the app that "may make things quite a lot noisier in logs (potentially contributing to a saturated server if you're sending them across the wire "

### CODE REVIEW

**Review by rhystmills — DISMISSED** [review]
Code looks sensible. Was able to restore a piece of content in CODE, and do the other things you'd want to be able to do (see a list snapshots, view the article JSON) - so looks good to me.

**Review by jonathonherbert — CHANGES_REQUESTED** [review]
@dskamiotis before we merge, can I recommend turning off logging unhandled error rejections, per this [conversation on chat?](https://chat.google.com/room/AAAAUqoLvh4/YbZZD7FbMuE)

Here's an example of how to do it on [stack overflow](https://stackoverflow.com/a/41993170) – happy to pair!

**Comment by rhystmills** [comment]
(Seems like I missed something important in chat, reverting approval for now)

File: public/javascripts/app/lib/btn/index.js [hunk 1]
```diff
@@ -3,8 +3,8 @@ import angular from 'angular';
 var btn = angular.module('guBtn', []);
 
 var templates = {
-  'anchor': '<a class="{{className}}" ng-transclude=true></a>',
-  'button': '<button class="{{className}}" ng-transclude=true></button>',
+  'anchor': '<a class="{{className}}" ng-transclude><!-- --></a>',
+  'button': '<button class="{{className}}" ng-transclude><!-- --></button>',
```
Comment (twrichards) [inline public/javascripts/app/lib/btn/index.js:7]: is the html comment needed?

**Comment by dskamiotis** [comment]
> @dskamiotis before we merge, can I recommend turning off logging unhandled error rejections, per this [conversation on chat?](https://chat.google.com/room/AAAAUqoLvh4/YbZZD7FbMuE)
> 
> Here's an example of how to do it on [stack overflow](https://stackoverflow.com/a/41993170) – happy to pair!

It appears `main.js` is the entry point for this app, hopefully this solves the over active logging on start up of app

**Review by jonathonherbert — APPROVED** [review]
Tested in CODE, navigating through versions and restoring them works as expected 👍

**Pull request merged**